### PR TITLE
TIP-1198 Don't need to check if web/bundle exist

### PR DESCRIPTION
**Pull request by arnolanglade**
**Description (for Contributor and Core Developer)**

This PR optimizes the class autoloading: we don't need to load classes in production if we only use them for test and there were some duplicated loading rules too.

Some post scripts are removed because we don't need them. Our install command do the same job, it is a pity to do two times the same job to install the PIM. We will save a little bit time to build our PIM :)

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | -
| Added legacy Behats               | -
| Added acceptance tests            | -
| Added integration tests           | -
| Changelog updated                 | -
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | -
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -85,6 +85,9 @@ jobs:
           - run:
                 name: Start containers
                 command: docker-compose up -d fpm
+          - run:
+                name: Generate production environment cache
+                command: docker-compose exec fpm bin/console cache:clear -e dev
```
Comment (skeleton) [inline .circleci/config.yml]: dev env?

Comment (mmetayer) [reply .circleci/config.yml]: yes, it's for phpstan

Comment (arnolanglade) [reply .circleci/config.yml]: Phpstan need a an xml file, this file is only available in dev env. you should have a look [that](https://github.com/phpstan/phpstan-symfony#configuration) to have more info.

Comment (skeleton) [reply .circleci/config.yml]: so you should change the name from "Generate production environment cache" to "Generate dev environment cache"

File: composer.json [hunk 2]
```diff
@@ -25,9 +28,7 @@
             "Pim\\Behat\\": "tests/legacy/features/Behat",
             "AkeneoTest\\": "tests/back",
             "Akeneo\\Test\\IntegrationTestsBundle\\": "tests/back/Integration/IntegrationTestsBundle/",
-            "Akeneo\\Test\\Integration\\": "tests/back/Integration/",
```
Comment (anaelChardan) [inline composer.json]: Wasn't it needed?

File: .circleci/config.yml (hunk 1 above)
Comment (anaelChardan) [reply .circleci/config.yml]: Same comment as @skeleton

Comment (arnolanglade) [reply .circleci/config.yml]: ah ok, didn't see that! If the CI is green I will merge this PR but I will change that in my next PR.

**Comment by ahocquard** [comment]
I have one question: is the merge with the standard planned in the future?

Generally, when you release in production, you rely on composer to initialize all the codebase you need on a dedicated server.
It's possible to not have access to composer anymore in a production environment., but only an archive.

If you prevent to use composer for the installation, it means that you can't prepare the codebase by only using composer, without having a database up, an ES up, etc...

For dev environment, we don't care, but for the install of the PIM in the future, I'm not sure. When we migrate a standard version, we don't call `pim:install`.

**Comment by arnolanglade** [comment]
First we plan to remove standard edition, we want to generate them. You should have a look to https://github.com/akeneo/pim-enterprise-dev/pull/6532
Then, you don't run composer in production but you need to package your application before.
Finally, We will work our tools to easily migrate because we plan to deliver the next major every month.

**Pull request merged**